themes/Night Owl-color-theme.json

```diff
@@ -1016,7 +1016,17 @@
       "scope": "meta.method.declaration storage.type.js",
       "settings": {
         "foreground": "#82AAFF",
-        "fontStyle": "normal"
+        "fontStyle": ""
```

Comment:
I removed this because `normal` is not a valid value. I was getting this warning `Font style must be 'italic', 'bold' or 'underline' or a combination or the empty string.`

Reply:
Ah yeah, I made this mistake for a number of things earlier and fixed it, I must've missed one! Thanks a lot.